dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-4.1/resources/l10n/com.ibm.websphere.appserver.microProfile-4.1.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature combines the Liberty features that support Eclipse MicroProfile 4.1 for Cloud Native Java.
```

Comment:
```suggestion
description=This feature combines the Liberty features that support MicroProfile 4.1 for Cloud Native Java.
```

Reply:
Do you want to remove the word Eclipse from all of these NLS files? For versions 4.1 and 5.0? Do you want to change the description of 4.0 as well? I think we should do this under a separate PR since it might need NLS review and possibly re-translation?

Reply:
I changed my mind, I'll add this change to this PR.